src/include/sof/audio/buffer.h

```diff
@@ -343,6 +345,33 @@ static inline void buffer_copy(struct comp_buffer *source,
 	}
 }
 
+static inline void buffer_set_params(struct comp_buffer *buffer, void *data)
```

Comment:
`void` because it can be something other than `sof_ipc_stream_params` in other situations where this callback is implemented?

Reply:
Yes, you are right, that make no sense, updated.